config_automation.yml

```diff
@@ -1,33 +1,39 @@
 
+### Render preview of content with changes (Rmd's and md's are checked)
+render-preview: true
+
 ##### Checks run at pull request #####
 # Check quiz formatting
-check-quizzes: no
+check-quizzes: false
+quiz_error_min: 0
 # Check that urls in the content are not broken
-url-checker: yes
-# Render preview of content with changes (Rmd's and md's are checked)
-render-preview: yes
+url-checker: true
+url_error_min: 0
 # Spell check Rmds and quizzes
-spell-check: yes
+spell-check: true
+spell_error_min: 0
+
+#### Other options
 # Style any R code
-style-code: yes
+style-code: true
 # Would you like your markdown files to be checked for formatting
-markdown-linter: yes
+markdown-linter: true
 # Would you like a readability report on your markdowns?
-readability-report: yes
+readability-report: true
 # Test build the docker image if any docker-relevant files have been changed
-docker-test: no
+docker-test: false
 # Should URLs be tested periodically?
-url-check-periodically: yes
+url-check-periodically: true
 
 ##### Renderings run upon merge to main branch #####
 # Rendering each platform's content
 render-website: rmd
-render-leanpub: yes
-render-coursera: no
```

Comment:
hmm interesting that this sais now to rendering coursera... I think we have quizzes in a private repo but need to check

Reply:
yes we do, hmmm strange that leanpub says yes though... and usually we would still use the no_toc